test/annotate_attribute.ll

```diff
@@ -0,0 +1,141 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o - | FileCheck %s --check-prefix=CHECK-SPIRV
+
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.rev.bc
+; RUN: llvm-dis < %t.rev.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; CHECK-SPIRV: Decorate {{[0-9]+}} UserSemantic "42"
+; CHECK-SPIRV: Decorate {{[0-9]+}} UserSemantic "bar"
+; CHECK-SPIRV: Decorate {{[0-9]+}} UserSemantic "{FOO}"
+; CHECK-SPIRV: MemberDecorate {{[0-9]+}} 1 UserSemantic "128"
+; CHECK-SPIRV: MemberDecorate {{[0-9]+}} 0 UserSemantic "{baz}"
+
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64-unknown-linux"
+
+%class.anon = type { i8 }
+%struct.bar = type { i32, i8 }
+
+; CHECK-LLVM: [[STR1:@[a-zA-Z0-9_.]+]] = {{.*}}42
```

Comment:
I think it will be easier to understand the test if you rename this variable to STR, in order to match its name with IR counterpart.